example/example-node.js

```diff
@@ -132,6 +132,76 @@ const htmlString =  `<!DOCTYPE html>
                 </td>
             </tr>
         </table>
+
+        <div class="page-break" style="page-break-after: always"></div>
```

Comment:
add this in examples/example.js and examples/react-example/src/App.js

Reply:
done